tests/test_apply.py

```diff
@@ -0,0 +1,105 @@
+# -*- coding: utf-8 -*-
+import copy
+import io
+import pathlib
+import shutil
+import tempfile
+from unittest import mock
+
+import numpy as np
+import pytest
+
+from eko import basis_rotation as br
+from eko import output
+from ekomark import apply
+
+from .test_output import FakePDF
+
+
+class TestApply:
```

Comment:
Ok - you can copy ;-) could you please find out if there is a way to inherit also in tests (which I assume there is)